composer/trainer/trainer.py

```diff
@@ -122,7 +124,7 @@ def __init__(
             *,
             model: BaseMosaicModel,
             train_dataloader: Union[DataLoader, DataSpec],
-            eval_dataloader: Union[DataLoader, DataSpec],
+            eval_dataloader: Union[DataLoader, DataSpec, List[Evaluator]],
```

Comment:
Type annotation is inconsistent with docstring. Can we support an evaluator or sequence of evaluators?